app/views/govuk_publishing_components/components/docs/step_by_step_nav.yml

```diff
@@ -194,7 +194,7 @@ examples:
       - style, an attribute on the parent list that controls its appearance. Defaults to optional but can be set to 'choice' (adds indent and bullets to the list)
       - context, an optional text field to show some extra information after the link, usually a monetary value
       - active, whether to make the link highlighted (to indicate the current page)
-      - highlight_step, a general option to highlight the step. Usually the active link would be in the active step, but in some circumstances there might be an active step but no highlighted link.
+      - `highlight_step`, a general option to highlight the step. Usually the active link would be in the active step, but in some circumstances there might be an active step but no highlighted link.
```

Comment:
If you're highlighting `highlight_step`, shouldn't `context` and `active` also be highlighted?